replica/table.cc

```diff
@@ -2994,10 +2994,7 @@ table::mutation_query(schema_ptr s,
 
     std::exception_ptr ex;
   try {
-    // Un-reverse the schema sent to the coordinator, it expects the
-    // legacy format.
```

Comment:
It may still expect it!

Reply:
Agree. When the coordinator is an old node the data is reversed first before sending it back: `storage_proxy::remote::handle_read` https://github.com/scylladb/scylladb/blob/b9f7e2f45dd4d20695f1f5f94fe68e42868c80d6/service/storage_proxy.cc#L819

Reply:
Aha